docs/using-airbyte/getting-started/readme.md

```diff
@@ -2,6 +2,10 @@
 products: all
```

Comment:
nit: can this one page be for oss and cloud only (exclude self-managed enterprise, which has it's own getting started page)

Reply:
I think since this is the getting started page for "using airbyte" while the Enterprise docs focus more on setting it up, I feel this page still applies to |Enterprise users as well. I feel it's def worth though putting a paragraph in here besides Cloud and OSS to link to the Enterprise setup page! cc @nataliekwong